test/cutting/test_cutting_roundtrip.py

```diff
@@ -179,3 +180,28 @@ def test_cutting_exact_reconstruction(example_circuit):
     logger.info("Max error: %f", np.max(np.abs(exact_expvals - simulated_expvals)))
 
     assert np.allclose(exact_expvals, simulated_expvals, atol=1e-8)
+
+
+def test_sampler_fail(example_circuit):
+    """This test checks if the sampler throws an error if you pass it a subscircuit with no measurements. Tests temporary workaround to Issue #422.
+
+    This test passes if no exceptions are raised.
+
+    """
+
+    qc = example_circuit
+    observable_to_test = PauliList(
+        ["IIZ"]
+    )  # Without the workaround to Issue #422, this throws a Sampler error.
```

Comment:
```suggestion
    )  # Without the workaround to Issue #422, this observable causes a Sampler error.
```
(because the error does not get thrown on _this_ line)